src/app/components/profesionales/formacionGrado/formacion-grado-detalle.component.ts

```diff
@@ -141,13 +155,36 @@ export class FormacionGradoDetalleComponent implements OnInit {
         this._profesionalService.putProfesional(this.profesional)
             .subscribe(resp => {
                 this.profesional = resp;
+                this.plex.toast('success', 'Se guardo con exito el verificar papeles!', 'informacion', 2000);
+            }, () => {
+                this.plex.toast('danger', 'Error en el guardado de papeles verificados!', 'informacion', 2000);
             });
 
     }
 
+    rechazarRenovacion() {
+        this.edicionRechazo = true;
+    }
+    opcionRechazarRenovacion() {
+        const op = (this.formacion.matriculacion && this.formacion.matriculado && !this.formacion.papelesVerificados && this.formacion.renovacionOnline
```

Comment:
const op = (this.formacion.matriculacion && this.formacion.matriculado  && this.formacion.renovacionOnline